src/Microsoft.AspNet.Authentication/DataHandler/Serializer/TicketSerializer.cs

```diff
@@ -41,21 +41,29 @@ public static void Write([NotNull] BinaryWriter writer, [NotNull] Authentication
             writer.Write(FormatVersion);
```

Comment:
Ah yeah, it will likely crash in the for loop, due to the way it's coded :+1: 

`for (int i = 0; i != identityCount; ++i)`